src/rules/unit-case/index.js

```diff
@@ -25,18 +25,11 @@ export default function (expectation) {
     if (!validOptions) { return }
 
     root.walkDecls(decl => {
-      const { value } = decl
-
-      valueParser(value).walk((node) => {
+      valueParser(decl.value).walk((node) => {
         // Ignore wrong units within `url` function
         if (node.type === "function" && node.value.toLowerCase() === "url") { return false }
```

Comment:
@davidtheclark Otherwise it will be a bit confusing. If i return null, we go inside function, but we don't want this.